osu.Game.Rulesets.Osu.Tests/TestSceneDrawableJudgement.cs

```diff
@@ -43,10 +43,8 @@ public void TestHitLightingDisabled()
             showResult(HitResult.Great);
 
             AddUntilStep("judgements shown", () => this.ChildrenOfType<TestDrawableOsuJudgement>().Any());
-            AddAssert("judgement body immediately visible",
-                () => this.ChildrenOfType<TestDrawableOsuJudgement>().All(judgement => judgement.JudgementBody.Alpha == 1));
             AddAssert("hit lighting hidden",
-                () => this.ChildrenOfType<TestDrawableOsuJudgement>().All(judgement => judgement.Lighting.Alpha == 0));
+                () => this.ChildrenOfType<TestDrawableOsuJudgement>().All(judgement => !judgement.Lighting.Transforms.Any()));
```

Comment:
this assertion change is a little far-reaching I'd say - I'd probably check `TargetMember` as well for a bit more robustness

Reply:
can actually be changed back if you'd prefer. the important part ended up being the All -> Any.